app/services/reblog_service.rb

```diff
@@ -50,12 +50,8 @@ def create_notification(reblog)
     end
   end
 
-  def bump_potential_friendship(account, reblog)
+  def bump_potential_friendship(_account, _reblog)
     ActivityTracker.increment('activity:interactions')
-
-    return if account.following?(reblog.reblog.account_id)
-
-    PotentialFriendshipTracker.record(account.id, reblog.reblog.account_id, :reblog)
```

Comment:
Same, this method should probably be removed (with the `ActivityTracker` call moved) or renamed, since it doesn't bump anything but the global interactions count.